app/blog/(posts)/1000-days-of-running/page.mdx

```diff
@@ -0,0 +1,173 @@
+
+But the problem with that type of loose training was that I didn&apos;t actually get _faster_.
+My max speed was still super slow, and my pace at shorter distances was still much slower than equivalent paces calculated from my half marathon pace.
+This feels super obvious in hindsight, but this remained unchanged until I started doing interval training and doing some track workouts.
+It took me a bit of time to figure out how to get comfortable deciding what paces, lengths, and reps to do.
+Like many other aspects of running, I just had to try things arbitrarily and see what worked.
+For example, you can imagine doing 12 <Distance m={400} /> reps at slightly-faster-than your <Distance m={5000} /> pace will slowly improve your speed at the <Distance m={5000} /> distance, so you can experiment with different paces and rest amounts.
+
+<Callout>
+  Interval training works because it lets you do targeted work at specific speeds, which trains your body to become more efficient at those speeds.
+  Track workouts work because they let you push as hard as you can, which increases your max speed.
+</Callout>
+
+By the end of the second year, my shorter distances had basically caught up.
+
+### 1000 days (100%) + future
+
+<Callout intent="none">
+  Trained to run faster, shorter distances.
+</Callout>
+
+I&apos;m now comfortable with every type of training (interval training, long runs, lactate threshold, easy runs, etc.), so I decide what to run each day based on what my goals are (while maintaining a base weekly mileage of <Distance m={144_841} />).
+Currently, I&apos;m focusing on the <Distance m={5_000} /> distance because it&apos;s what I find fun right now, I want to build up more speed while I&apos;m still young, and I&apos;m kinda close to breaking a huge barrier (14:59, PB is a 15:49).
+
+The long-term goal is still to get good at running the half marathon/marathon distances&mdash;it would be really cool to qualify for the US Olympic Trials (requires a ~<Pace s={196} /> marathon as of 2025), but I&apos;m not really seriously thinking about it yet because it&apos;s still a long way off.
+
+## Learnings
+
+Beyond the stuff I obviously learned directly about running (e.g. how to run, recover, etc.), here&apos;s a list of things I learned that I thought were interesting:
+- People seem to think I am more disciplined and have better work ethic after finding out I run every day.
+- Running discipline doesn&apos;t translate at all to other aspects of life.
+- Most people don&apos;t even try when they say they run, and they get way too swamped with preconceptions on what they _should_ learn, self-reflect upon, or gain from running, but you can kind of tell they don&apos;t actually feel it.
```

Comment:
It's kind of redundant (a bit cocky) for this to point. 
It's not really a learning, and I think it's better off not saying (personal opinion)
It's rare that you're already doing the stuff you're doing and let other people get inspiration from it, imo.